### ISSUE TITLE
where is the pretrained model

### ISSUE THREAD

**Issue by lgcy** [fh2019ustc/DocTr-Plus#1]

**Comment by Coder-C18** [comment]
I also want to know "where is the pretrained model"

**Comment by fh2019ustc** [comment]
Hi, I am sorry for the late reply. You can try our [online demo](https://doctrp.docscanner.top/).

**Comment by Coder-C18** [comment]
it is wonderful,i hope in the future you can public pre-train mode

**Comment by wujushan** [comment]
+1

**Comment by lengyueyang** [comment]
+1

**Comment by jinhopark-lomin** [comment]
+1

**Comment by dedkamaroz** [comment]
+5

**Comment by THE-FLIER** [comment]
+6

**Comment by keepfoolisher** [comment]
hello , i have release my first test version model base on this model, just for fun, @lgcy @dedkamaroz @wujushan @Coder-C18 @jinhopark-lomin @THE-FLIER @lengyueyang

**Comment by Juddd** [comment]
> hello , i have release my first test version model base on this model, just for fun, @lgcy @dedkamaroz @wujushan @Coder-C18 @jinhopark-lomin @THE-FLIER @lengyueyang

Doesn't look like illumination rectification?

**Comment by keepfoolisher** [comment]
> > hello , i have release my first test version model base on this model, just for fun, @lgcy @dedkamaroz @wujushan @Coder-C18 @jinhopark-lomin @THE-FLIER @lengyueyang
> 
> Doesn't look like illumination rectification?

yes,  its just for rectification,  illumination is another function and its used for my employer，maybe i will try authors method and release a illumination rectification version at free time.

**Comment by StaelTchinda** [comment]
+7

**Comment by Luomiou** [comment]
+8

**Comment by lerndeep** [comment]
+9

**Comment by zhaopu0710** [comment]
+9

**Comment by decadance-dance** [comment]
+11

**Comment by decadance-dance** [comment]
@fh2019ustc, real cool job and I'm impressed by the demo.
Is there any special condition under which you can release weights?
I'm interested in understanding whether we should even hope that the weights will soon be publicly available?

**Comment by probably-harish** [comment]
> > > hello , i have release my first test version model base on this model, just for fun, [@lgcy](https://github.com/lgcy) [@dedkamaroz](https://github.com/dedkamaroz) [@wujushan](https://github.com/wujushan) @Coder-C18 [@jinhopark-lomin](https://github.com/jinhopark-lomin) [@THE-FLIER](https://github.com/THE-FLIER) [@lengyueyang](https://github.com/lengyueyang)
> > 
> > 
> > Doesn't look like illumination rectification?
> 
> yes, its just for rectification, illumination is another function and its used for my employer，maybe i will try authors method and release a illumination rectification version at free time.

@keepfoolisher is there a direct download link for the model weights? and how did you train the model, can you help me with the training data also
